source/dub/recipe/packagerecipe.d

```diff
@@ -169,8 +169,10 @@ struct BuildSettingsTemplate {
 		if (!this.targetName.empty) dst.targetName = this.targetName;
 		if (!this.workingDirectory.empty) dst.workingDirectory = this.workingDirectory;
 		if (!this.mainSourceFile.empty) {
-			dst.mainSourceFile = this.mainSourceFile;
-			dst.addSourceFiles(this.mainSourceFile);
+			auto p = NativePath(this.mainSourceFile);
+			p.normalize();
+			dst.mainSourceFile = p.toNativeString();
+			dst.addSourceFiles(p.toNativeString());
```

Comment:
I doubt it matters but the result of `toNativeString` could be cached.

Reply:
I'll simply use `dst.mainSourceFile`.

BTW, would be good to get #1409 in, so that this one can be rebased on top before adding a change log entry.